index.html

```diff
@@ -1625,9 +1625,10 @@ <h3 id="6_E_4">6.E.4 Example</h3>
   "data": {
     "search_results": {
       "shipments": [{
-        "shipment_identifier" : "FOO-222-BAR-42",
+        "reference_identifier" : "FOO-222-BAR-42",
         "recipient": {
           "cc": "NL",
+          "region": "Zuid-Holland",

```

Comment:
Dit werkt nog niet live. Welke card hoort hierbij? Misschien dit in een nieuwe pull-request staan; want region moet op meerdere plekken worden toegevoegd.